Bump up python version python:alpine3.17

ShaYmez opened the pull request:
Bump up python version to alpine 3.17 (Stable)

ShaYmez committed:
Update Dockerfile

Upgrade python version to 3.10

ShaYmez committed:
Update Dockerfile

Bump up python version to the latest. alpine 3.17

nreisbeck commented:
👍🏻

ShaYmez committed:
Rename .gitignore to gitignoreHASH

ignore gitnore while we build docker from actions

M0VUB committed:
Add workflow for docker-build

ShaYmez committed:
Update and rename gitignoreHASH to .gitignore

.gitignore add workflows

ShaYmez committed:
Update docker-compose.yml

Open 20 ports by default

ShaYmez commented:
Updated docker image shaymez/hblink3:latest to support multi arch - includes arm64/arm/v7/amd64/386. No updates to base code.

nreisbeck commented:
> Updated docker image shaymez/hblink3:latest to support multi arch - includes arm64/arm/v7/amd64/386. No updates to base code.

@ShaYmez do you need some maintenance assistance? Pushing (overwriting tags) is a pretty bad practice, I'd be willing to offer my help to get a functional release system with code tags and docker image tags set up, but continually pushing `:latest` is incredibly bad practice.

ShaYmez committed:
Update docker-compose.yml

Include /json/ enviroment outside container so that we can manage those iffy dumps from RadioID

ShaYmez committed:
Update Dockerfile

Bump up python version and force pip to use latest release

ShaYmez committed:
Update Dockerfile

Fix bug in Dockerfile

ShaYmez committed:
Update Dockerfile

Purge cache

ShaYmez committed:
Create playback-SAMPLE.cfg

Add sample playback config for the parrot system

ShaYmez committed:
Update hblink-SAMPLE.cfg

Add parrot into configuration stanza & update version

ShaYmez committed:
Update .gitignore

ShaYmez committed:
Update rules_SAMPLE.py

Add playback parrot example stanzas into rules

ShaYmez committed:
Update docker-compose.yml

Add variable for enabling parrot (if needed)

ShaYmez committed:
Update docker-compose.yml

Disable by default =0 enable =1

ShaYmez committed:
Update entrypoint

Change entrypoint to include playback enviroment variable

ShaYmez committed:
Update entrypoint

Typo

ShaYmez committed:
Rename playback-SAMPLE.cfg to playback.cfg

Bring playback cfg into play ;)

ShaYmez committed:
Update hblink-SAMPLE.cfg

Update config and fix logging paths

ShaYmez committed:
Update playback.cfg

Fix logging paths and add to console

ShaYmez committed:
Update hblink-SAMPLE.cfg

Set parrot to False by default

ShaYmez committed:
Update playback.cfg

Set parrot logging to console only

ShaYmez committed:
Update docker-compose.yml

Map ports for masters outside the container!

ShaYmez committed:
Update docker-compose.yml

Fix port ranges ahh!

ShaYmez commented:
@nreisbeck Thanks. Sounds good! im no expert on deployment, I do this to make life easier.. but happy to help and absorb your knowlege too :)

ShaYmez committed:
Update playback.cfg

Add 9990 to the mix

ShaYmez committed:
Create version.txt

Add versioning

ShaYmez committed:
Update version.txt

Versioning - rebuild image - update

Copilot committed:
Initial plan

Copilot committed:
Fix typos, remove commented code, and simplify boolean comparisons

Co-authored-by: ShaYmez <76499782+ShaYmez@users.noreply.github.com>

ShaYmez committed:
Merge pull request #1 from ShaYmez/copilot/analyse-and-review

Code quality improvements: fix typos, remove dead code, simplify boolean comparisons

Copilot committed:
Initial plan

Copilot committed:
Fix Debian 13 compatibility: Update Alpine base, fix shell script, update GitHub Actions

Co-authored-by: ShaYmez <76499782+ShaYmez@users.noreply.github.com>

Copilot committed:
Update dependency versions for better Debian 13 compatibility and remove obsolete docker-compose version

Co-authored-by: ShaYmez <76499782+ShaYmez@users.noreply.github.com>

Copilot committed:
Add CHANGELOG.md to document Debian 13 compatibility updates

Co-authored-by: ShaYmez <76499782+ShaYmez@users.noreply.github.com>

ShaYmez committed:
Merge pull request #2 from ShaYmez/copilot/update-debian-compatibility

Debian 13 compatibility: Update Alpine 3.20, Python dependencies, and CI workflows

ShaYmez committed:
Refactor Dockerfile for improved build process

ShaYmez committed:
Refactor entrypoint script for improved execution

Copilot committed:
Initial plan

Copilot committed:
Update version to 2.0.2 and finalize changelog for stable release

Co-authored-by: ShaYmez <76499782+ShaYmez@users.noreply.github.com>

ShaYmez committed:
Merge pull request #3 from ShaYmez/copilot/ready-release-2-0-2

Prepare stable release 2.0.2

ShaYmez committed:
Refactor hblink3 service configuration in docker-compose
